samples/recognize.js

```diff
@@ -263,7 +263,7 @@ async function asyncRecognizeGCS(
   const transcription = response.results
     .map(result => result.alternatives[0].transcript)
     .join('\n');
-  console.log(`Transcription: ${transcription}`);
+  console.log('Transcription: ${transcription}');
```

Comment:
this would print the string "Transaction: ${transcription}", rather than the string "Transcription transcription results".

Curious what issue you were bumping into with linting?